industrial_ci/src/tests/source_tests.sh

```diff
@@ -163,16 +163,29 @@ ici_time_start catkin_build
 
 # for catkin
 if [ "${TARGET_PKGS// }" == "" ]; then export TARGET_PKGS=`catkin_topological_order ${TARGET_REPO_PATH} --only-names`; fi
+# fall-back to all packages
+if [ "${TARGET_PKGS// }" == "" ]; then export TARGET_PKGS=`catkin_topological_order $CATKIN_WORKSPACE/src --only-names`; fi
```

Comment:
So this line and previous line is the "new behavior (regarding `TARGET_PKGS`)" in https://github.com/ros-industrial/industrial_ci/pull/256#issuecomment-356976994 . I'd never figure out by myself why these very similar 2 lines are needed. For future maintenance, would you add a comment? Just add a link is fine IMO.

Reply:
Addressed.